src/colorprint.cc

```diff
@@ -168,7 +168,8 @@ bool IsColorTerminal() {
       "xterm",         "xterm-color",     "xterm-256color",
       "screen",        "screen-256color", "tmux",
       "tmux-256color", "rxvt-unicode",    "rxvt-unicode-256color",
-      "linux",         "cygwin",
+      "linux",         "cygwin",          "xterm-kitty",
```

Comment:
Please update the URL to point at https://github.com/google/googletest/blob/v1.13.0/googletest/src/gtest.cc#L3225-L3259